addons/mail/static/src/new/activity/activity_service.js

```diff
@@ -7,6 +7,7 @@ export const activityService = {
     start(env, { orm, action, bus_service: bus }) {
         const state = reactive({
             counter: 0,
+            feedback: {},
```

Comment:
I would have been best to have a proper data structure for store of activities, something like:
```js
activities: { <activityId>: <activity> }
```

That way feedback is clearly inside the proper activity, rather than in another place of store and it's synced with same key `activityId`.

Would need to code clean a bit activity service for that.